src/App.tsx

```diff
@@ -589,64 +575,53 @@ function App() {
             ref={resultRef}
           >
             {
               <>
                 <h3>
-                  Your transaction status on {messageDisplay.l2Network.name}
+                  L2{" "}
+                  {messageDisplay.l2TxHash !== "null" ? (
+                    <ExternalLink
+                      href={`${messageDisplay.explorerUrl}/tx/${messageDisplay.l2TxHash}`}
+                    >
+                      Transaction
+                    </ExternalLink>
+                  ) : (
+                    "Transaction"
+                  )}{" "}
+                  status on {messageDisplay.l2Network.name}
                 </h3>
+                {messageDisplay.l1ToL2Message !== undefined && (
+                  <ExternalLink
+                    href={`${
+                      messageDisplay.explorerUrl
+                    }/tx/${getRetryableIdOrDepositHash(messageDisplay)}`}
+                    showIcon
+                  >
+                    Check Your Retryable Ticket on{" "}
+                    {messageDisplay.l2Network.name}
+                  </ExternalLink>
+                )}
                 <p>{messageDisplay.text}</p>
-                {messageDisplay.showRedeemButton ? (
+                {messageDisplay.showRedeemButton && (
                   <Redeem
                     l1ToL2Message={messageDisplay}
                     signer={signer}
                     connectedNetworkId={chain?.id}
                   />
-                ) : null}
+                )}
               </>
             }
-            <p>
-              -----Txn links----- <br />
-              {
-                <>
-                  {messageDisplay.l1ToL2Message !== undefined ? (
-                    <a
-                      href={
-                        messageDisplay.explorerUrl +
-                        "/tx/" +
-                        getRetryableIdOrDepositHash(messageDisplay)
-                      }
-                      rel="noreferrer"
-                      target="_blank"
-                    >
-                      Retryable Ticket
-                    </a>
-                  ) : null}
-                </>
-              }
-              <br />
-              {messageDisplay.l2TxHash !== "null" && (
-                <>
-                  <a
-                    href={
-                      messageDisplay.explorerUrl +
-                      "/tx/" +
-                      messageDisplay.l2TxHash
-                    }
-                    rel="noreferrer"
-                    target="_blank"
-                  >
-                    L2 Tx
-                  </a>
-                  <br />
-                </>
-              )}
-            </p>
           </div>
         );
       })}
+      {(messagesDisplays.some((msg) => msg.showRedeemButton) ||
+        l2ToL1MessagesToShow.some(
+          (msg) => msg.status === L2ToL1MessageStatus.CONFIRMED
+        )) && <ConnectButtons />}
```

Comment:
likewise w/ https://github.com/OffchainLabs/retryable-tx-panel/pull/29/files#r1072429010

Reply:
can try with this one:
https://retryable-tx-panel-git-styling-offchain-labs.vercel.app/tx/0x3b82e7582ec91694997f2dee2c030bbc0c149d98acd3503f4b55a50de20c3cf3